salt/modules/virtualenv_mod.py

```diff
@@ -335,16 +333,22 @@ def get_distribution_path(venv, distribution):
     return ret['stdout']
 
 
-def get_resource_path(venv, package_or_requirement, resource_name):
+def get_resource_path(venv, package=None, resource=None, package_or_requirement=None, resource_name=None):
```

Comment:
@srkunze, will you move these new args to the end of the list to minimize API change?  Thanks.